### ISSUE TITLE
The first item in tree views should be focused on first focus

### ISSUE THREAD

**Issue by Neurrone** [microsoft/vscode#97816]
Issue Type: <b>Bug</b>

CC: @isidorn

1. Cause a tree view to appear. For example, hit ctrl+shift+f, perform a search and press enter. Then hit tab 5 times until the tree view gains focus.

Since this will be the first time that the tree view gains focus, it would be nice if the first item in the tree automatically gained focus, instead of having to press e.g, down arrow once to trigger focus inside.

VS Code version: Code - Insiders 1.46.0-insider (288852d8c26d797974bfb8aed06e1bff9b809223, 2020-05-12T05:43:31.363Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz (8 x 1992)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.86GB (3.64GB free)|
|Process Argv||
|Screen Reader|yes|
|VM|33%|
</details><details><summary>Extensions (13)</summary>

Extension|Author (truncated)|Version
---|---|---
notes|can|1.0.0
vscode-markdownlint|Dav|0.35.1
vscode-eslint|dba|2.1.5
prettier-vscode|esb|4.7.0
rest-client|hum|0.23.2
python|ms-|2020.4.76186
remote-wsl|ms-|0.44.2
cpptools|ms-|0.28.0
Go|ms-|0.14.1
vsnotes|pat|0.7.1
rust|rus|0.7.8
markdown-preview-enhanced|shd|0.5.3
vscode-webhint|web|1.5.5

</details>

**Comment by isidorn** [comment]
@Neurrone thanks for feedback. This is actually how the lists and trees have always behaved. Changing this might be breaking behavior for our users.
Though I do understand that we could potentialy change the following: the very first time a tree is focused, focus the first element. Pressing esc would still clear focus and focus the whole tree.

Though not sure. Assigning also to @joaomoreno to think about this

**Comment by Neurrone** [comment]
> Though I do understand that we could potentialy change the following: the very first time a tree is focused, focus the first element. Pressing esc would still clear focus and focus the whole tree.

Yup, I was referring specifically to the very first focus. Basically to remove the papercut where the extra keystroke is required to start getting spoken feedback / start navigating. If the user presses escape to unset focus, then that should be preserved when tabbing away then back.

**Comment by joaomoreno** [comment]
I'd say this is specific behavior per tree/list.

**Comment by Neurrone** [comment]
Wondering why this was closed?

**Comment by isidorn** [comment]
@neurrone we closed it because we have no plans to tackle it in the near future.
This milestone we are going through old issues and figuring out which are the ones we want to focus on.
